raaghu-layouts/src/rds-comp-app-shell/rds-comp-app-shell.stories.tsx

```
@@ -338,55 +916,563 @@
+              label: "Menus",
+              path: "",
+            },
+            {
+              icon: "newsletters",
+              key: "5",
+              label: "Newsletters",
+              path: "",
+            },
+            {
+              icon: "pages",
+              key: "5",
```

ESLint

[error] Expected indentation of 28 spaces but found 14.

(indent)